GrandCentralBoard/Utilities/GoogleAPIDataProvider.swift

```diff
@@ -30,12 +30,8 @@ final class GoogleAPIDataProvider {
 
     private func refreshTokenOperation() -> Operation {
         let refreshTokenOperation = BlockOperation(block: { [weak self] (continueWithError) in
-            guard let strongSelf = self else {
-                continueWithError(error: nil)
-                return
-            }
 
-            strongSelf.tokenProvider.accessTokenFromRefreshToken({ result in
+            self?.tokenProvider.accessTokenFromRefreshToken({ result in
```

Comment:
I'm not sure if this change is correct. If `self == nil` then we'll never call `continueWithError` and if I understand correctly, the operation will never finish, right?